aws-rds-dbinstance/src/main/java/software/amazon/rds/dbinstance/validators/OracleCustomSystemId.java

```diff
@@ -0,0 +1,21 @@
+package software.amazon.rds.dbinstance.validators;
+
+import software.amazon.rds.common.request.RequestValidationException;
+import software.amazon.rds.dbinstance.ResourceModel;
+import software.amazon.rds.dbinstance.util.ResourceModelHelper;
+
+public class OracleCustomSystemId {
+    public static void validateRequest(final ResourceModel model) throws RequestValidationException {
+        if (ResourceModelHelper.isRestoreToPointInTime(model) && model.getDBSystemId() != null) {
```

Comment:
You'll probably have to add tests where `model.getDBSystemId()` resolves to `false` in order to pass the coverage.